tools/Directory.Build.props

```diff
@@ -2,6 +2,11 @@
 
     <Import Project="../Directory.Build.props" />
 
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp3.0</TargetFramework>

```

Comment:
not strictly necessary, but I figured we'd get better annotations from the libraries, like knowing what `IsNullOrEmpty` means for its arguments.

Reply:
:+1: